README.md

```diff
@@ -118,240 +118,228 @@ To build and compile ISIS3 requires following the instructions listed below, whi
 
+**Mars Exploration Rover Mission (kernels can be excluded):**
 
-<span id="MarsGlobalSurveyorMission"></span>
+    cd $ISIS3DATA
+    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/mer .
 
-Mars Global Surveyor Mission (kernels can be excluded):
 
-    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/mgs data/
+**Mariner10 Mission (kernels can be excluded):**
 
+    cd $ISIS3DATA
+    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/mariner10 .
 
-<span id="MarsReconnaissanceOrbiterMission"></span>
 
-Mars Reconnaissance Orbiter Mission (kernels can be excluded):
+**Messenger Mission (kernels can be excluded):**
 
-    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/mro data/
+    cd $ISIS3DATA
+    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/messenger .
 
 
-<span id="MarsOdysseyMission"></span>
+**Mars Express Mission (kernels can be excluded):**
 
-Mars Odyssey Mission (kernels can be excluded):
+    cd $ISIS3DATA
+    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/mex .
 
-    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/odyssey data/
 
+**Mars Global Surveyor Mission (kernels can be excluded):**
 
-<span id="NearMission"></span>
+    cd $ISIS3DATA
+    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/mgs .
 
-Near Mission (kernels can be excluded):
 
-    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/near data/
+**Mars Reconnaissance Orbiter Mission (kernels can be excluded):**
 
+    cd $ISIS3DATA
+    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/mro .
 
-New Horizons Mission (kernels can be excluded):
 
-<span id="NewHorizonsMission"></span>
+**Mars Odyssey Mission (kernels can be excluded):**
 
-    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/newhorizons data/
+    cd $ISIS3DATA
+    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/odyssey .
 
 
-Odyssey Mission (kernels can be excluded):
+**Near Mission (kernels can be excluded):**
 
-<span id="OdysseyMission"></span>
+    cd $ISIS3DATA
+    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/near .
 
-            rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/odyssey data/
 
+**New Horizons Mission (kernels can be excluded):**
 
-Rolo Mission (kernels can be excluded):
+    cd $ISIS3DATA
+    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/newhorizons .
 
-<span id="RoloMission"></span>
 
-    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/rolo data/
+**Odyssey Mission (kernels can be excluded):**
 
+    rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isis3data/data/odyssey .
```

Comment:
This one is missing a `cd $ISIS3DATA`

Reply:
And actually, "Mars Odyssey" is already on this list -- maybe this whole section should just be removed?

Reply:
Should be fixed now

Reply:
@jessemapel :+1: